WD-Firmware/examples/demo/demo_app.h

```diff
@@ -0,0 +1,68 @@
+/*

```

Comment:
do rename to main->rtosal_demo.c so we can keep history there.
main.c will be new